1.10/deploying-services/containerizers/ucr.md

```diff
@@ -5,7 +5,7 @@ feature_maturity: preview
 menu_order: 10
 ---
 
-The [Universal Container Runtime](http://mesos.apache.org/documentation/latest/container-image) (UCR) extends the Mesos containerizer to support provisioning [Docker](https://docker.com/) container images ([AppC](https://github.com/appc/spec) coming soon). This means that you can use both the Mesos containerizer and other container image types in DC/OS. You can still use the Docker container runtime directly with DC/OS, but the Universal Container Runtime supports running Docker images without depending on the Docker Engine, which allows for better integration with Mesos.
+The [Universal Container Runtime](http://mesos.apache.org/documentation/latest/container-image) (UCR) extends the Mesos containerizer to support provisioning [Docker](https://docker.com/) container images. This means that you can use both the Mesos containerizer and other container image types in DC/OS. You can still use the Docker container runtime directly with DC/OS, but the Universal Container Runtime supports running Docker images without depending on the Docker Engine, which allows for better integration with Mesos.
```

Comment:
>  which allows for better integration with Mesos.

it isn't about better integration... it is about:
1. reducing dependencies
2. reducing service downtime (upgrading or restarting docker kills all running containers on a node)
3. improving on the fly upgradability